R/pkg/inst/profile/shell.R

```diff
@@ -29,5 +30,18 @@
   sqlContext <- SparkR::sparkRSQL.init(sc)
   assign("sqlContext", sqlContext, envir=.GlobalEnv)
   cat("\n Welcome to SparkR!")
```

Comment:
We can remove `SparkR` from here and just make this `Welcome to`. That way the entire sentence will read `Welcome to Spark`